configure.ac

```diff
@@ -4513,26 +4513,30 @@ AC_ARG_WITH(lazy_gc_thread_creation, [  --with-lazy-gc-thread-creation=yes|no
 	fi
 ], [with_lazy_gc_thread_creation=no])
 
-AC_ARG_WITH(cooperative_gc, [  --with-cooperative-gc=yes|no      Enable cooperative stop-the-world garbage collection (sgen only) (defaults to no)], [], [with_cooperative_gc=default])
+AC_ARG_WITH(cooperative_gc,      [  --with-cooperative-gc=yes|no           Enable cooperative stop-the-world garbage collection (sgen only) (defaults to no)], [], [with_cooperative_gc=default])
+AC_ARG_WITH(cooperative_suspend, [  --with-cooperative-suspend=yes|no      Enable cooperative stop-the-world garbage collection (sgen only) (defaults to no)], [], [with_cooperative_suspend=default])
```

Comment:
should we mark the `--with-cooperative-gc` option as deprecated and recommend the new one?

Reply:
How do we do that? Just put in the message "(deprecated)"?

Reply:
Yes. Just so that someone reading this in 6 months doesn't wonder why we have two flags that do the same.

Reply:
I added a `AC_MSG_WARN` for `--with-cooperative-gc`